### PR TITLE
[Term Entry] Lua Math .abs()

### PR DESCRIPTION
**Pull request by parthrc**
### Description

Added the math.abs() term for Lua
Closes issue #3385 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/lua/concepts/mathematical-library/terms/abs/abs.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
+Title: 'abs()'
+Description: 'Converts given number to absolute value'
```
Comment (caupolicandiaz) [inline content/lua/concepts/mathematical-library/terms/abs/abs.md:3]:
```suggestion
Description: 'Returns the absolute value of a given number.'
```

File: content/lua/concepts/mathematical-library/terms/abs/abs.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  - 'Game Development'
+Tags:
+  - 'Math'
+  - 'Functions'
+CatalogContent:
+  - 'learn-lua'
+  - 'paths/computer-science'
+---
+
+The **`abs()`** function in Lua's math library converts a number  to its absolute value.
```
Comment (caupolicandiaz) [inline content/lua/concepts/mathematical-library/terms/abs/abs.md:15]:
```suggestion
The **`abs()`** function in Lua's math library returns the absolute value of a given number.
```

File: content/lua/concepts/mathematical-library/terms/abs/abs.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+## Syntax
+
+```pseudo
+math.abs(number)
+```
+
+- `number`: The number.
```
Comment (caupolicandiaz) [inline content/lua/concepts/mathematical-library/terms/abs/abs.md:23]:
```suggestion
- `number`: The number to be addressed.
```

File: content/lua/concepts/mathematical-library/terms/abs/abs.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+## Example
+
+In this example, `math.abs()` converts -5 to 5.
```
Comment (caupolicandiaz) [inline content/lua/concepts/mathematical-library/terms/abs/abs.md:27]:
```suggestion
This example demonstrates a basic implementation of `math.abs()`.
```

Comment (caupolicandiaz) [reply content/lua/concepts/mathematical-library/terms/abs/abs.md:27]: Including the output here is redundant

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@parthrc I've added my comments. Please update the entry, and tag me when you've finished making the changes. Thanks

**Comment by parthrc** [comment]
Hey @caupolicandiaz done with changes.

**Review by caupolicandiaz — APPROVED** [review]
@parthrc This is ready for a second review

**Review by SSwiniarski — APPROVED** [review]
@parthrc Looks good.

**Pull request merged**